packages/manager/src/lib/getSentryEnvironmentInfo.ts

```diff
@@ -0,0 +1,32 @@
+import semver from "semver";
+
+type PackageManifest = { name: string; version: string };
+
+type SentryEnvironmentInfo = { environment: string; release: string };
+
+export const getSentryEnvironmentInfo = (
```

Comment:
Is this information specific to Sentry? What do you think of making this more generic, like `getEnvironmentInfo()`?

If you think this function will evolve to eventually be Sentry specific, we can keep this name. 🙂